### ISSUE TITLE
[Android][Accessibility] [On multiple input validation failures the card scrolls out of viewport]

### ISSUE THREAD

**Issue by dipja** [microsoft/AdaptiveCards#5019]
# Platform

What platform is your issue or question related to? (Delete other platforms).

- [ ] Android

# Author or host

Microsoft Teams

# Version of SDK

2.3.0

# Details

On multiple input validation failures on the same card (multiple taps on submit action) sdk tried to clear the focus first and then request the focus onto the invalid input field. This is leading to card scrolling out of the view port.

[Video ](https://microsoft-my.sharepoint.com/:v:/p/dipja/Ea2Vr3TyeZBPlm_pL55oUh4BEVIpc0aXieF3c2om6E6F-A?e=KwgBWW)of the experience.

**Comment by golddove** [comment]
In recyclerview with multiple cards, clear focus right before setting focus causes unpredictable scrolling. This does not repro in our visualizer, but does repro in the sample `mobilechatapp`.

Plan: remove clear focus step entirely, which will fix this issue. But, this causes less than ideal behavior in some scenarios, which we will track as a possible bug (#5027).
